### PR TITLE
Tapestry

### PR DESCRIPTION
**Pull request by fortmarek**
### Short description 📝

Publishing new release seems to have become quite a tedious tasks with a lot of steps - that's why I think it'd be good to automate these steps with [tapestry](https://github.com/ackeecz/tapestry).

### Solution 📦
Adding tapestry to the project.

### Implementation 👩‍💻👨‍💻

Configuring `TapestryConfig` with release steps from `RELEASE.md` - now release process has 3 steps, instead of 10 (no automation for interacting with Github release just yet).

### CODE REVIEW

File: Tapestries/Sources/TapestryConfig/TapestryConfig.swift [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import PackageDescription
+
+let config = TapestryConfig(release: Release(actions: [.pre(.dependenciesCompatibility([.cocoapods, .spm(.all)])),
```
Comment (pepibumur) [inline Tapestries/Sources/TapestryConfig/TapestryConfig.swift]: Out of curiosity, why should the path be `Tapestries/Sources/TapestryConfig/TapestryConfig.swift`? Would it make sense to shorten it?

Comment (fortmarek) [reply Tapestries/Sources/TapestryConfig/TapestryConfig.swift]: Not really - to edit `TapestryConfig.swift` with autocompletion and all, I need a way to include `Tapestry` library; I found creating a standalone `tapestries` package away from the main project to be the least intrusive. And this standalone package is why I need to follow the SPM folder structure.

Comment (pepibumur) [reply Tapestries/Sources/TapestryConfig/TapestryConfig.swift]: Oh I see. In that case all good :+1:.

File: RELEASE.md [hunk 2]
```diff
@@ -2,12 +2,6 @@
...
-2. Update Carthage dependencies if they are outdated with `bundle exec rake carthage_update_dependencies`.
-3. Validate the state of the project by running `bundle exec rake release_check`.
-4. Update the `CHANGELOG.md` adding a new entry at the top with the next version. Make sure that all the changes in the version that is about to be released are properly formatted.
-5. Update the version in the `xcodeproj.podspec` and `README.md` files.
-7. Commit, tag and push the changes to GitHub.
-8. Create a new release on [GitHub](https://github.com/tuist/XcodeProj) including the information from the last entry in the `CHANGELOG.md`.
-9. Push the changes to CocoaPods: `bundle exec pod trunk push --allow-warnings --verbose`.
-10. Archive the Carthage framework by running `bundle exec rake arhive_carthage` and attach the `XcodeProj.framework.zip` to the GitHub release.
+1. Run `tapestry release version-number` (eg `tapestry release 7.1.0`) *(Install [tapestry](https://github.com/ackeecz/tapestry) and [tuist](https://github.com/tuist/tuist) if you don't have them installed already)*.
```
Comment (pepibumur) [inline RELEASE.md]: @fortmarek will Tapestry update the CHANGELOG to include the new section? In essence what used to be the step 4.

Comment (fortmarek) [reply RELEASE.md]: Yes, it finds `## Next` and replaces it with 
```
## Next

## Version-number
```

Comment (pepibumur) [reply RELEASE.md]: Nice :+1:. Well handled!

Comment (fortmarek) [reply RELEASE.md]: It also updates the version in `README.md` and in `.podspec`. I ran a trial run locally and it seemed fine.

**Pull request merged**